<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton_autoroll">
          <property name="text">
           <string>自动摇点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_s">
            <item>
             <property name="text">
              <string>无所谓</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≥</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≤</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_s">
            <property name="value">
             <number>40</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>CON</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_c">
            <item>
             <property name="text">
              <string>无所谓</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≥</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≤</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_c">
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>INT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_i">
            <item>
             <property name="text">
              <string>无所谓</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≥</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≤</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_i">
            <property name="value">
             <number>40</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>WIS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_w">
            <item>
             <property name="text">
              <string>无所谓</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≥</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≤</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_w">
            <property name="value">
             <number>15</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>AGI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_a">
            <item>
             <property name="text">
              <string>无所谓</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≥</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>≤</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_a">
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_roll">
        <property name="text">
         <string>摇点</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_s">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_c">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>CON</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_i">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>INT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_w">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>WIS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_a">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>AGI</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_str">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_con">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_int">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_wis">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_agi">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>注入，只需点击一次</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>by Shantom</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
